### ISSUE TITLE
Add warning if property contains the default value

### ISSUE THREAD

**Issue by OTTOxMarc** [SAP/ui5-language-assistant#377]
Hello, I have a feature request for you:

I sometimes find myself copying source code from the samples page. These examples are sometimes "bloated" (e.g. `SimpleForm`). Or I just try different values for a specific property and then stick with the default value because I like it the most (e.g. different `backgroundDesign`s for a `Panel`. 

It would be nice to know that I can just remove the property because it contains the default value.

**Comment by bd82** [comment]
Hello @OTTOxMarc 

I think that such a feature would have a pre-requirement of "user configurable validation rules" capability first.
Because this is basically a **style** preference, and we cannot always show "Style" warnings, particularly if there are many of them.

**Comment by OTTOxMarc** [comment]
Yes, I see that this might not be important for everyone and should be (de)activatable in some kind of user settings.
